openapi-cli/src/test/resources/generators/schema/ballerina/schema_with_request_body_ref.bal

```diff
@@ -14,10 +14,3 @@ public type Dog record {
     *Pet;
     boolean bark?;
 };
-
```

Comment:
Why this `Pet` record is not generating now ? `Pet` is used in multiple places too

Reply:
It is a mistake by copying the generated code from resources  `Pet` is there. I updated the resource file